ios/RNIapIosSk2.m

```diff
@@ -3,14 +3,11 @@
 #import <React/RCTBridgeModule.h>
 #ifdef __IPHONE_15_0
 
-// From: https://stackoverflow.com/a/5337804/570612
-#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v)  ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] != NSOrderedAscending)
-
 
 @interface RCT_EXTERN_MODULE (RNIapIosSk2, NSObject)
 
 RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(isAvailable){
-    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"15.0")) {
+    if (@available(iOS 15.0, *)) {
```

Comment:
I've refactored this to use `@available` the `SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO` was returning "1.0" for visionOS and `@available` is universal. 

Source: https://roadfiresoftware.com/2014/01/the-right-way-to-check-the-ios-version-in-an-app/